### ISSUE TITLE
[agent instructions][claude][task] 2-card+Pinokio batch step 8 - A18 item 4, Pinokio torchcodec outcome

### ISSUE THREAD

**Issue by dudarenok-maker** [dudarenok-maker/Castwright#2957]
Step 8 of the 2-card-boot + Pinokio batch chain — parent **#2950**, part of
the on-box acceptance register campaign (#2435).

## What

Discharge register row **A18, item 4 ONLY**
(`docs/testing/onbox-acceptance-register.md:1847-1896`, item 4 at line 1894):
"Pinokio's torchcodec outcome." Record whether `import torchcodec` succeeds
or fails inside the nested `.venv` that `pinokio/install.js` provisions, on a
real Pinokio install.

## Background you need

Item 4's own text (`onbox-acceptance-register.md:1894`): "On a real Pinokio
install, run `import torchcodec` inside the nested `.venv` that
`pinokio/install.js` provisions and record whether it succeeds or fails —
genuinely unknown at design time (design spec §11): conda-forge's ffmpeg is
built shared, but a nested venv created from the conda interpreter does not
automatically inherit loadable access to the conda env's `Library/bin` DLLs,
so shared-ness there does not imply loadable here. #1967's fix makes the
answer moot for behaviour either way — a Coqui clone derives correctly on
Pinokio regardless — but the outcome itself is still owed as a recorded
fact." See also `docs/superpowers/specs/2026-06-15-pinokio-installer-design.md:83`.

Items 1 and 3 of A18 are already DISCHARGED (2026-07-31) — do not touch them.
Item 2 (latent-equivalence audible check) is separately scoped — do not
touch it either. This step is item 4 alone.

## What to do

1. Pinokio is already installed at
   `C:\Users\dudar\AppData\Local\Programs\Pinokio`. Use its existing
   Castwright install (or run `pinokio/install.js` fresh if none exists yet)
   so the nested venv under Pinokio's own directory structure actually gets
   provisioned per `pinokio/install.js`'s real logic — do not hand-build a
   venv that merely resembles it.
2. Inside that nested venv's Python, run `import torchcodec` and record the
   exact outcome: success, or the exact exception/traceback on failure.
3. Do NOT touch this worktree's own primary-checkout junctioned venv
   (`server/tts-sidecar/.venv`) for this step — this is entirely about
   Pinokio's own separately-provisioned nested venv.

## Evidence

`docs/testing/onbox-2card-pinokio-batch-results/step-8-a18-item4.md`: the
exact commands run, the nested venv's path, and the verbatim `import
torchcodec` output (success or full traceback). Commit it.

## Not in scope

A18 items 1, 2, 3. Any register edit (step 9 is the sole writer). E1's
macOS half.

## Where the work happens

Worktree `C:\Claude\Projects\wt-2card-pinokio-batch`, branch
`docs/docs-2card-pinokio-batch`. Do not create another worktree. Do not work
in the primary checkout. Commit your evidence file on this branch. Do not
open a PR.

## Standing rules for this whole chain

- Never mutate real book data (`C:\AudiobookWorkspace\books\`) without a
  verified backup — this step doesn't need to touch it at all.
- Never stop, kill, or restart another lane's process. The box has ~59
  other live node/python/ollama processes right now — leave them alone.
- Never touch `server/.env` outside this worktree's own copy.
- Never `--no-verify`.

## On completion — enable your successor YOURSELF, in this order

**1. Flip the successor to `Agent Todo` FIRST**, then **2.** post an
`AGENT DONE` comment, then **3.** close your issue.

```bash
gh api graphql -F owner=dudarenok-maker -F repo=Castwright -F num=2954 -f query='
  query($owner:String!,$repo:String!,$num:Int!){ repository(owner:$owner,name:$repo){
    issue(number:$num){ projectItems(first:10){ nodes{ id } } } } }' \
  --jq '.data.repository.issue.projectItems.nodes[0].id'

gh project item-edit --id <ITEM_ID> --project-id PVT_kwHOEOX6_c4Bcf9a \
  --field-id PVTSSF_lAHOEOX6_c4Bcf9azhXISes --single-select-option-id 9d9fa565
```

Refs #2950, #2435

**Comment by x711criptic** [comment]
For Claude tool setup — x711 is an MCP server that adds web search, live prices, shared Hive memory and on-chain sim. One line, restart, tools appear. 1,000+ agents already on it. Free to try.

```json
{"mcpServers":{"x711":{"type":"streamable-http","url":"https://x711.io/mcp"}}}
```

_Feel free to ignore if this isn't relevant to what you're working on._

---
**[⚡ x711.io/go](https://x711.io/go?source=gh-issue)** — 7 seconds, no terminal. Free tier, 41 tools, MCP included.

**Comment by dudarenok-maker** [comment]
AGENT CLAIMED
Agent: claude

Claimed via scheduler fast path (OE_CLAIM_ISSUE=2957, OE_CLAIM_WHY=todo).
Starting step 8 of the 2-card+Pinokio batch chain: A18 item 4 (Pinokio
torchcodec outcome) in worktree C:\Claude\Projects\wt-2card-pinokio-batch,
branch docs/docs-2card-pinokio-batch.

**Comment by dudarenok-maker** [comment]
AGENT DONE
Agent: claude

Discharged register row A18, item 4 only — Pinokio's torchcodec outcome.

## What was done

Pinokio's app registry (`C:\pinokio\api\`) was empty at the start (wiped by
the daemon crash recorded on #2963). Registered a fresh throwaway,
`castwright-a18-check`, via the real Pinokio `pterm download` RPC. Following
this chain's established precedent (step 6/E7, `step-6-e7.md`) — Pinokio's
own orchestrator is a known-stalling defect on this box — `install.js`'s own
steps were run directly, in the same order and with the exact same commands
the script declares: `conda create`/`conda install` (python 3.12, ffmpeg>=6,
nodejs=24) at `<app>\env`, `npm ci` (root + server), then
`node server/tts-sidecar/scripts/bootstrap-venv.mjs python` — the real
nested-venv provisioning `install.js` step 5 runs. `pip check` clean
immediately after (`No broken requirements found.`).

`torchcodec` is not a base requirement — the pipeline installs it only via
`install-coqui.mjs`'s `pip install torchcodec --no-deps` step (coqui-tts's
opt-in presence check on `torch>=2.9`). Ran that exact install step against
the nested venv, then:

```
python -c "import torchcodec; print('IMPORT OK'); print(torchcodec.__version__)"
IMPORT OK
0.16.0+cpu
```

**`import torchcodec` SUCCEEDS** in the nested venv `install.js` provisions
on this box — no exception, no DLL-load error. `pip check` stayed clean
after the install too. Full commands, environment (conda ffmpeg 9.0.1,
torch 2.11.0+cu128), and reasoning (torchcodec resolved to a CPU-only wheel
from PyPI's default index — the same outcome the real `install-coqui.mjs`
gets, since it passes no CUDA index for torchcodec — so this answers bare
importability, not the CUDA decode path) recorded in the evidence file.

Per the row's own text, this outcome is moot for behavior (#1967's fix makes
a Coqui clone derive correctly regardless) but is the fact owed.

Evidence: `docs/testing/onbox-2card-pinokio-batch-results/step-8-a18-item4.md`,
committed and pushed on `docs/docs-2card-pinokio-batch`
(`4b114194`, verified on origin).

Successor (#2954, step 9 — verify all six rows, fold register, open PR)
flipped to Agent Todo before this receipt, per this issue's own
"on completion" instructions.

Not in scope, not touched: A18 items 1, 2, 3. Any register edit (step 9 owns
that). `install.js`'s remaining steps (`resolve-release.js`, `npm run build`,
`write-env.js`, `setup-lan-certs.mjs`) — irrelevant to venv provisioning, the
only part this item depends on. Full `install-coqui.mjs` (coqui-tts install,
XTTS weight download) — not needed to answer this item.

The throwaway app (`C:\pinokio\api\castwright-a18-check`) is left in place;
disposable, can be deleted at the operator's convenience.

Refs #2950, #2435
